internal/pkg/cosign/tsa/mock/mock_tsa_client.go

```diff
@@ -117,5 +117,5 @@ func (c *TSAClient) GetTimestampResponse(tsq []byte) ([]byte, error) {
 		tsStruct.Time = c.Time
 	}
 
-	return tsStruct.CreateResponse(c.CertChain[0], c.Signer)
+	return tsStruct.CreateResponseWithOpts(c.CertChain[0], c.Signer, hashAlg)
```

Comment:
These are different hash algs, this is for generating the hash for the signed timestamp structure. Let’s just hardcode to sha256

Reply:
got it, thanks